Win11ThemeTest/ComboBoxTest.cs

```diff
@@ -128,36 +131,39 @@ public void comboBox2_editableText()
 
         //Check whether the dropdown is clickable or not.
         [Test]
-        public void comboBox3_mouseClick()
+        public void ComboBox3_mouseClick()
         {
-            Assert.IsNotNull(comboBox);
-            Mouse.MoveTo(comboBox.GetClickablePoint());
-            Mouse.Click();
+            Assert.That(comboBox, Is.Not.Null);
+            comboBox.Click();
             Assert.That(comboBox.ExpandCollapseState, Is.EqualTo(ExpandCollapseState.Expanded));
             comboBox.Collapse();
         }
 
         [Test]
-        public void comboBox4_mouseSelectClick()
+        public void ComboBox4_mouseSelectClick()
         {
-            Assert.IsNotNull(comboBox);
+            Assert.That(comboBox, Is.Not.Null);
             Mouse.MoveTo(comboBox.GetClickablePoint());
-            Mouse.Click();            
-            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(500));           
+            Mouse.Click();
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(500));
             comboBox.Items[3].Click();
-            Assert.That(comboBox.SelectedItem.Name, Is.EqualTo("Yellow"));                    
+            Assert.That(comboBox.SelectedItem.Name, Is.EqualTo("Yellow"));
         }
 
         [Test]
-        public void comboBox5_cleanUp()
+        public void ComboBox5_cleanUp()
         {
-            Assert.IsNotNull(comboWindow);
-            comboWindow.Close();
-            Wait.UntilInputIsProcessed();
-            Assert.IsTrue(comboWindow.IsOffscreen);
-            Assert.IsNotNull(mainWindow);
-            mainWindow.Close();
-            Assert.IsTrue(mainWindow.IsOffscreen);
+            if (app != null)
+            {
+                app.Close();
+                Console.WriteLine("Application closed successfully.");
+                Assert.That(app.Close());
+            }
+            else
+            {
+                Console.WriteLine("Application not found.");
+                Assert.That(app.Close());
```

Comment:
Here if app is null then we cannot close the app - app.close() will throw exception. Consider this in all the other controls as well.